src/mongocxx/write_concern.hpp

```diff
@@ -52,8 +52,6 @@ class uri;
 ///
 /// @see http://docs.mongodb.org/manual/core/write-concern/
 ///
-/// @todo this interface is terrible -- it's exactly what you get with the c-driver essentially.
```

Comment:
hey everyone, I don't recall why I thought there was room for improvement, forge ahead!